### ISSUE TITLE
Build fails for Homebrew PHP7 on OSX since v4.0.3

### ISSUE THREAD

**Issue by vearutop** [tideways/php-profiler-extension#17]
Build is failing for `edfb29e` and `v4.0.3`
```
12:30:16 ~/php-profiler-extension master ls | grep -v .git | xargs rm -rf
12:30:27 ~/php-profiler-extension master git reset --hard
HEAD is now at edfb29e Bump version to 4.0.4
12:30:33 ~/php-profiler-extension master phpize
Configuring for:
PHP Api Version:         20151012
Zend Module Api No:      20151012
Zend Extension Api No:   320151012
12:30:38 ~/php-profiler-extension master ./configure
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for a sed that does not truncate output... /usr/bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-apple-darwin15.3.0
checking host system type... x86_64-apple-darwin15.3.0
checking target system type... x86_64-apple-darwin15.3.0
checking for PHP prefix... /usr/local/Cellar/php70/7.0.4
checking for PHP includes... -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/Cellar/php70/7.0.4/lib/php/extensions/no-debug-non-zts-20151012
checking for PHP installed headers prefix... /usr/local/Cellar/php70/7.0.4/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.14.3 (ok)
checking for gawk... gawk
checking whether to enable Tideways support... yes, shared
checking for clock_gettime... no
checking for clock_gettime in -lrt... no
checking for clock_get_time... yes
checking PHP version... PHP 7.x
checking for ld used by cc... /Library/Developer/CommandLineTools/usr/bin/ld
checking if the linker (/Library/Developer/CommandLineTools/usr/bin/ld) is GNU ld... no
checking for /Library/Developer/CommandLineTools/usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 196608
checking command to parse /usr/bin/nm output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking if cc supports -fno-rtti -fno-exceptions... yes
checking for cc option to produce PIC... -fno-common
checking if cc PIC flag -fno-common works... yes
checking if cc static flag -static works... no
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/Library/Developer/CommandLineTools/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin15.3.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
12:30:48 ~/php-profiler-extension master make
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=compile cc  -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /Users/vpoturaev/php-profiler-extension/php7/spans.c -o php7/spans.lo
mkdir php7/.libs
 cc -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /Users/vpoturaev/php-profiler-extension/php7/spans.c  -fno-common -DPIC -o php7/.libs/spans.o
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=compile cc  -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /Users/vpoturaev/php-profiler-extension/tideways.c -o tideways.lo
mkdir .libs
 cc -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /Users/vpoturaev/php-profiler-extension/tideways.c  -fno-common -DPIC -o .libs/tideways.o
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: warning: using the result of an assignment as a condition without
      parentheses [-Wparentheses]
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
            ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: note: place parentheses around the assignment to silence this warning
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
                     ^
            (                                                                       )
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: note: use '==' to turn this assignment into an equality comparison
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
                     ^
                     ==
/Users/vpoturaev/php-profiler-extension/tideways.c:734:3: warning: incompatible pointer types initializing 'zval *'
      (aka 'struct _zval_struct *') with an expression of type 'zval **' (aka 'struct _zval_struct **'); dereference with *
      [-Wincompatible-pointer-types]
                ZVAL_COPY_VALUE(&params[0], context);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_types.h:850:9: note: expanded from macro 'ZVAL_COPY_VALUE'
                zval *_z1 = (z);                                                                \
                      ^     ~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:744:20: warning: incompatible pointer types assigning to 'zval *'
      (aka 'struct _zval_struct *') from 'zval ***' (aka 'struct _zval_struct ***') [-Wincompatible-pointer-types]
                twcb->fci.params = (zval ***)params;
                                 ^ ~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:1249:3: warning: incompatible pointer types passing 'zval **'
      (aka 'struct _zval_struct **') to parameter of type 'zval *' (aka 'struct _zval_struct *'); dereference with *
      [-Wincompatible-pointer-types]
                zval_ptr_dtor(&retval_ptr);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_variables.h:122:48: note: expanded from macro 'zval_ptr_dtor'
#define zval_ptr_dtor(zval_ptr) _zval_ptr_dtor((zval_ptr) ZEND_FILE_LINE_CC)
                                               ^~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_variables.h:112:36: note: passing argument to parameter 'zval_ptr' here
ZEND_API void _zval_ptr_dtor(zval *zval_ptr ZEND_FILE_LINE_DC);
                                   ^
4 warnings generated.
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=link cc -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -o tideways.la -export-dynamic -avoid-version -prefer-pic -module -rpath /Users/vpoturaev/php-profiler-extension/modules  php7/spans.lo tideways.lo
cc ${wl}-flat_namespace ${wl}-undefined ${wl}suppress -o .libs/tideways.so -bundle  php7/.libs/spans.o .libs/tideways.o
duplicate symbol _hp_globals in:
    php7/.libs/spans.o
    .libs/tideways.o
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [tideways.la] Error 1
12:30:52 ~/php-profiler-extension master
```
However it works fine for `v4.0.0-beta2`
```
12:32:09 ~/php-profiler-extension master ls | grep -v .git | xargs rm -rf
12:32:14 ~/php-profiler-extension master git reset --hard
HEAD is now at edfb29e Bump version to 4.0.4
12:32:19 ~/php-profiler-extension master git checkout v4.0.0-beta2
Note: checking out 'v4.0.0-beta2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 276b35f... Disable doctrine.query support for now.
12:32:22 ~/php-profiler-extension (v4.0.0-beta2) phpize
Configuring for:
PHP Api Version:         20151012
Zend Module Api No:      20151012
Zend Extension Api No:   320151012
12:32:39 ~/php-profiler-extension (v4.0.0-beta2) ./configure
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for a sed that does not truncate output... /usr/bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-apple-darwin15.3.0
checking host system type... x86_64-apple-darwin15.3.0
checking target system type... x86_64-apple-darwin15.3.0
checking for PHP prefix... /usr/local/Cellar/php70/7.0.4
checking for PHP includes... -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/Cellar/php70/7.0.4/lib/php/extensions/no-debug-non-zts-20151012
checking for PHP installed headers prefix... /usr/local/Cellar/php70/7.0.4/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.14.3 (ok)
checking for gawk... gawk
checking whether to enable Tideways support... yes, shared
checking for clock_gettime... no
checking for clock_gettime in -lrt... no
checking for clock_get_time... yes
checking for ld used by cc... /Library/Developer/CommandLineTools/usr/bin/ld
checking if the linker (/Library/Developer/CommandLineTools/usr/bin/ld) is GNU ld... no
checking for /Library/Developer/CommandLineTools/usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 196608
checking command to parse /usr/bin/nm output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking if cc supports -fno-rtti -fno-exceptions... yes
checking for cc option to produce PIC... -fno-common
checking if cc PIC flag -fno-common works... yes
checking if cc static flag -static works... no
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/Library/Developer/CommandLineTools/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin15.3.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
12:32:48 ~/php-profiler-extension (v4.0.0-beta2) make
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=compile cc  -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /Users/vpoturaev/php-profiler-extension/tideways.c -o tideways.lo
 cc -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /Users/vpoturaev/php-profiler-extension/tideways.c  -fno-common -DPIC -o .libs/tideways.o
/Users/vpoturaev/php-profiler-extension/tideways.c:1072:15: warning: using the result of an assignment as a condition without
      parentheses [-Wparentheses]
        if (zidx_ptr = zend_hash_str_find(hp_globals.span_cache, summary, strlen(summary))) {
            ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:1072:15: note: place parentheses around the assignment to silence this warning
        if (zidx_ptr = zend_hash_str_find(hp_globals.span_cache, summary, strlen(summary))) {
                     ^
            (                                                                             )
/Users/vpoturaev/php-profiler-extension/tideways.c:1072:15: note: use '==' to turn this assignment into an equality comparison
        if (zidx_ptr = zend_hash_str_find(hp_globals.span_cache, summary, strlen(summary))) {
                     ^
                     ==
/Users/vpoturaev/php-profiler-extension/tideways.c:1155:3: warning: incompatible pointer types initializing 'zval *'
      (aka 'struct _zval_struct *') with an expression of type 'zval **' (aka 'struct _zval_struct **'); dereference with *
      [-Wincompatible-pointer-types]
                ZVAL_COPY_VALUE(&params[0], context);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_types.h:850:9: note: expanded from macro 'ZVAL_COPY_VALUE'
                zval *_z1 = (z);                                                                \
                      ^     ~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:1164:20: warning: incompatible pointer types assigning to 'zval *'
      (aka 'struct _zval_struct *') from 'zval ***' (aka 'struct _zval_struct ***') [-Wincompatible-pointer-types]
                twcb->fci.params = (zval ***)params;
                                 ^ ~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:1428:7: warning: assigning to 'char *' from 'const char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
        view = hp_get_base_filename(Z_STRVAL_P(name));
             ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:2684:15: warning: implicit declaration of function 'zend_exception_get_default' is
      invalid in C99 [-Wimplicit-function-declaration]
        default_ce = zend_exception_get_default(TSRMLS_C);
                     ^
/Users/vpoturaev/php-profiler-extension/tideways.c:2684:13: warning: incompatible integer to pointer conversion assigning to
      'zend_class_entry *' (aka 'struct _zend_class_entry *') from 'int' [-Wint-conversion]
        default_ce = zend_exception_get_default(TSRMLS_C);
                   ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:3491:3: warning: incompatible pointer types assigning to 'char *' from
      'zend_string *' (aka 'struct _zend_string *') [-Wincompatible-pointer-types]
                ZEND_HASH_FOREACH_KEY_VAL(ht, idx, str, val) {
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_hash.h:800:7: note: expanded from macro 'ZEND_HASH_FOREACH_KEY_VAL'
        _key = _p->key; \
             ^ ~~~~~~~
7 warnings generated.
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=link cc -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -o tideways.la -export-dynamic -avoid-version -prefer-pic -module -rpath /Users/vpoturaev/php-profiler-extension/modules  tideways.lo
cc ${wl}-flat_namespace ${wl}-undefined ${wl}suppress -o .libs/tideways.so -bundle  .libs/tideways.o
dsymutil .libs/tideways.so || :
creating tideways.la
(cd .libs && rm -f tideways.la && ln -s ../tideways.la tideways.la)
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=install cp ./tideways.la /Users/vpoturaev/php-profiler-extension/modules
cp ./.libs/tideways.so /Users/vpoturaev/php-profiler-extension/modules/tideways.so
cp ./.libs/tideways.lai /Users/vpoturaev/php-profiler-extension/modules/tideways.la
----------------------------------------------------------------------
Libraries have been installed in:
   /Users/vpoturaev/php-profiler-extension/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the `-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the `DYLD_LIBRARY_PATH' environment variable
     during execution

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------

Build complete.
Don't forget to run 'make test'.

12:32:53 ~/php-profiler-extension (v4.0.0-beta2)
```

And if you try to build `master` after `v4.0.0-beta2` configure, it will succeed.
```
12:34:48 ~/php-profiler-extension (v4.0.0-beta2) git checkout master
Previous HEAD position was 276b35f... Disable doctrine.query support for now.
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
12:35:03 ~/php-profiler-extension master make clean
find . -name \*.gcno -o -name \*.gcda | xargs rm -f
find . -name \*.lo -o -name \*.o | xargs rm -f
find . -name \*.la -o -name \*.a | xargs rm -f
find . -name \*.so | xargs rm -f
find . -name .libs -a -type d|xargs rm -rf
rm -f libphp.la       modules/* libs/*
12:35:08 ~/php-profiler-extension master make
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=compile cc  -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /Users/vpoturaev/php-profiler-extension/tideways.c -o tideways.lo
mkdir .libs
 cc -I. -I/Users/vpoturaev/php-profiler-extension -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /Users/vpoturaev/php-profiler-extension/tideways.c  -fno-common -DPIC -o .libs/tideways.o
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: warning: using the result of an assignment as a condition without
      parentheses [-Wparentheses]
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
            ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: note: place parentheses around the assignment to silence this warning
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
                     ^
            (                                                                       )
/Users/vpoturaev/php-profiler-extension/tideways.c:572:15: note: use '==' to turn this assignment into an equality comparison
        if (zidx_ptr = zend_hash_str_find(TWG(span_cache), summary, strlen(summary))) {
                     ^
                     ==
/Users/vpoturaev/php-profiler-extension/tideways.c:734:3: warning: incompatible pointer types initializing 'zval *'
      (aka 'struct _zval_struct *') with an expression of type 'zval **' (aka 'struct _zval_struct **'); dereference with *
      [-Wincompatible-pointer-types]
                ZVAL_COPY_VALUE(&params[0], context);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_types.h:850:9: note: expanded from macro 'ZVAL_COPY_VALUE'
                zval *_z1 = (z);                                                                \
                      ^     ~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:744:20: warning: incompatible pointer types assigning to 'zval *'
      (aka 'struct _zval_struct *') from 'zval ***' (aka 'struct _zval_struct ***') [-Wincompatible-pointer-types]
                twcb->fci.params = (zval ***)params;
                                 ^ ~~~~~~~~~~~~~~~~
/Users/vpoturaev/php-profiler-extension/tideways.c:1249:3: warning: incompatible pointer types passing 'zval **'
      (aka 'struct _zval_struct **') to parameter of type 'zval *' (aka 'struct _zval_struct *'); dereference with *
      [-Wincompatible-pointer-types]
                zval_ptr_dtor(&retval_ptr);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_variables.h:122:48: note: expanded from macro 'zval_ptr_dtor'
#define zval_ptr_dtor(zval_ptr) _zval_ptr_dtor((zval_ptr) ZEND_FILE_LINE_CC)
                                               ^~~~~~~~~~
/usr/local/Cellar/php70/7.0.4/include/php/Zend/zend_variables.h:112:36: note: passing argument to parameter 'zval_ptr' here
ZEND_API void _zval_ptr_dtor(zval *zval_ptr ZEND_FILE_LINE_DC);
                                   ^
4 warnings generated.
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=link cc -DPHP_ATOM_INC -I/Users/vpoturaev/php-profiler-extension/include -I/Users/vpoturaev/php-profiler-extension/main -I/Users/vpoturaev/php-profiler-extension -I/usr/local/Cellar/php70/7.0.4/include/php -I/usr/local/Cellar/php70/7.0.4/include/php/main -I/usr/local/Cellar/php70/7.0.4/include/php/TSRM -I/usr/local/Cellar/php70/7.0.4/include/php/Zend -I/usr/local/Cellar/php70/7.0.4/include/php/ext -I/usr/local/Cellar/php70/7.0.4/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -o tideways.la -export-dynamic -avoid-version -prefer-pic -module -rpath /Users/vpoturaev/php-profiler-extension/modules  tideways.lo
cc ${wl}-flat_namespace ${wl}-undefined ${wl}suppress -o .libs/tideways.so -bundle  .libs/tideways.o
dsymutil .libs/tideways.so || :
creating tideways.la
(cd .libs && rm -f tideways.la && ln -s ../tideways.la tideways.la)
/bin/sh /Users/vpoturaev/php-profiler-extension/libtool --mode=install cp ./tideways.la /Users/vpoturaev/php-profiler-extension/modules
cp ./.libs/tideways.so /Users/vpoturaev/php-profiler-extension/modules/tideways.so
cp ./.libs/tideways.lai /Users/vpoturaev/php-profiler-extension/modules/tideways.la
----------------------------------------------------------------------
Libraries have been installed in:
   /Users/vpoturaev/php-profiler-extension/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the `-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the `DYLD_LIBRARY_PATH' environment variable
     during execution

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------

Build complete.
Don't forget to run 'make test'.

12:35:14 ~/php-profiler-extension master sudo make install
Password:
Installing shared extensions:     /usr/local/Cellar/php70/7.0.4/lib/php/extensions/no-debug-non-zts-20151012/
12:36:18 ~/php-profiler-extension master php -i | grep tideways
Additional .ini files parsed => /usr/local/etc/php/7.0/conf.d/ext-tideways.ini,
tideways
tideways => 4.0.4
Connection (tideways.connection) => unix:///var/run/tideways/tidewaysd.sock
UDP Connection (tideways.udp_connection) => 127.0.0.1:8135
Default API Key (tideways.api_key) =>
Default Sample-Rate (tideways.sample_rate) => 30
Framework Detection (tideways.framework) =>
Automatically Start (tideways.auto_start) => Yes
Tideways Collect Mode (tideways.collect) => tracing
Tideways Monitoring Mode (tideways.monitor) => basic
Allowed Distributed Tracing Hosts (tideways.distributed_tracing_hosts) => 127.0.0.1
Load PHP Library (tideways.auto_prepend_library) => Yes
12:36:29 ~/php-profiler-extension master
```
So, probably `./configure` script was broken since `v4.0.0-beta2`.
Thanks.

**Comment by beberlei** [comment]
Yes this is also discussed in #15, i am already investigating.

**Comment by beberlei** [comment]
Fixed in a1a840f
